### ISSUE TITLE
EDGEDB_SERVER_ADMIN_UI​ env var is ignored

### ISSUE THREAD

**Issue by Arrow7000** [edgedb/edgedb#5386]
- EdgeDB Version: 2.14+7aec755
- EdgeDB CLI Version: 2.3.1+ef99779
- OS Version: Ubuntu 20.04.6 LTS (GNU/Linux 5.4.0-147-generic x86_64)

Steps to Reproduce:

1. Add `EDGEDB_SERVER_ADMIN_UI​=enabled` into `/etc/edgedb/env` on server
2. Run `systemctl restart edgedb.service`
3. The server still rejects the `edgedb ui` command, saying `the specified EdgeDB server is not serving Web UI.`

However when I changed the `$args` in `/etc/edgedb/start.sh` to include `--admin-ui=enabled` it did start serving the web UI. So it looks like despite the env var and the `edgedb-server` flags allegedly both doing the same thing, the env var appears to be ignored.

I spent a bit of time on a call with @fmoor earlier today who could confirm that changing the env var did nothing, while adding the flag fixed it.

**Comment by elprans** [comment]
Hm, is it documented somewhere that the server unit should read `/etc/edgedb/env`?  The expected way is to use a standard systemd drop-in, e.g:

```
systemctl edit edgedb-server-2.service
...
# Add Environment=EDGEDB_SERVER_ADMIN_UI=enabled to the [Service] section
...
systemctl restart edgedb-server-2.service
```

**Comment by fmoor** [comment]
The digital ocean image sets [`EnvironmentFile=/etc/edgedb/env`](https://github.com/edgedb/edgedb-deploy/blob/a93050f920301d62af21357241b42ad6919b599b/digitalocean-1-click/setup.sh#L110) to make the [deployment guide](https://github.com/edgedb/edgedb-deploy/blob/a93050f920301d62af21357241b42ad6919b599b/digitalocean-1-click/setup.sh#L110) more copy paste able.

I could not reproduce this behavior.

**Comment by elprans** [comment]
@Arrow7000, are you using the DigitalOcean droplet?

**Comment by Arrow7000** [comment]
@elprans yes I am. Using this image https://marketplace.digitalocean.com/apps/edgedb
